ingest/defaults/config.yaml

```
@@ -4,6 +4,9 @@
 # Define optional config parameters with their default values here so that users
 # do not have to dig through the workflows to figure out the default values
 
+# Required to fetch from Entrez
+entrez_search_term: "Mumps+orthorubulavirus[Organism]"
```

Suggestion: query by taxonomic identifier and [Primary Organism](https://www.ncbi.nlm.nih.gov/books/NBK49540/)

```suggestion
entrez_search_term: "taxid2560602[Primary Organism]"
```

This gives the same results but is more aligned with the query for NCBI Datasets, which is nice if we are using both.

---

Thanks for adjusting the search field! I did a quick skim of the [slack thread](https://bedfordlab.slack.com/archives/C7SDVPBLZ/p1745860369895809?thread_ts=1745525976.246009&cid=C7SDVPBLZ) discussing this but let me know if I missed a key point in applying your suggestion.

---

Ah, this search term is returning no results. Perhaps I'm missing something here @victorlin

---

Typo "tax" -> "tx" found it, nevermind

---

Whoops sorry about that!